storage/innobase/xtrabackup/test/t/pxb-1961.sh

```diff
@@ -0,0 +1,65 @@
+############################################################################
+# PXB-1961: xtraback fails with encryption='n' with Percona server
+############################################################################
+
+function backup_and_restore() {
+record_db_state test
+rm -rf $topdir/backup
+mkdir -p $topdir/backup
+# Backup
+xtrabackup --datadir=$mysql_datadir --backup --target-dir=$topdir/backup 
+# Prepare
+xtrabackup --datadir=$mysql_datadir --prepare --target-dir=$topdir/backup
+stop_server
+# Restore
+rm -rf $mysql_datadir/ibdata1 $mysql_datadir/ib_logfile* \
+    $mysql_datadir/test/*.ibd
+cp -r $topdir/backup/* $mysql_datadir
+start_server
+# Verify backup
+verify_db_state test
+
+}
+if ! is_debug_server  ; then
+       skip_test "Require debug server version"
+fi
+
+. inc/common.sh
+
+MYSQLD_EXTRA_MY_CNF_OPTS="
+innodb_file_per_table
+"
+
+. inc/keyring_file.sh
+
+start_server
+
+vlog "case#1 restore table created with encryption='n'"
+mysql test <<EOF
+set global innodb_page_cleaner_disabled_debug=on;
```

Comment:
Dont we need have_debug_server.inc?
Also what about disabling checkpoint?

Reply:
that is present in common.sh

Reply:
why did we put in common.sh? I think we need a separate .sh/.inc for have_debug_server

Reply:
in pxb all the methods are defined in common.sh like require_xtradb, require_server_version_higher_than, require_debug_pxb_version.

Reply:
the problem then is if a test that needs xtradb, will unnecessarily force them to be debug? :thinking:

Reply:
these are just functions. It works differently from mtr.

+function is_debug_server()
+{
+
+[[ $MYSQL_VERSION =~ .*debug ]]
+
+}
+

Reply:
ah, we source them. Then you are supposed to call is_debug_server() in the test? and skip it?

Reply:
I see. Did the review for 5.7